mypy_django_plugin/transformers/querysets.py

```diff
@@ -437,42 +474,32 @@ def extract_prefetch_related_annotations(ctx: MethodContext, django_context: Dja
                     continue
 
                 if lookup_value := helpers.resolve_string_attribute_value(lookup_expr, django_context):
-                    if django_model := helpers.get_model_info_from_qs_ctx(ctx, django_context):
```

Comment:
We were already doing this fetch in the guard branch at the top of this function